curvefs/src/mds/mds_service.cpp

```diff
@@ -253,11 +254,24 @@ void MdsServiceImpl::ListClusterFsInfo(
     ::curvefs::mds::ListClusterFsInfoResponse* response,
     ::google::protobuf::Closure* done) {
     brpc::ClosureGuard guard(done);
+    LOG(INFO) << "start to check cluster fs info.";
     fsManager_->GetAllFsInfo(response->mutable_fsinfo());
     LOG(INFO) << "ListClusterFsInfo success, response: "
               << response->ShortDebugString();
     return;
 }
 
+void MdsServiceImpl::StatMetadataUsage(
+    ::google::protobuf::RpcController* controller,
+    const ::curvefs::mds::StatMetadataUsageRequest* request,
+    ::curvefs::mds::StatMetadataUsageResponse* response,
+    ::google::protobuf::Closure* done) {
+    brpc::ClosureGuard guard(done);
+    LOG(INFO) << "start to state metadata usage.";
+    fsManager_->GetMetaServersSpace(response->mutable_metadatausages());
+    LOG(INFO) << "state metadata usage end.";
```

Comment:
You can move interface 'StatMetadataUsageRequest' from mds.proto to topology.proto and deal with directly without pass requst through fsManager.